source/tit/core/meta/set.hpp

```diff
@@ -47,6 +50,19 @@ class Set final {
     return index_of_v<U, Ts...>;
   }
 
+  /// Call a function with all elements.
+  template<std::invocable<Ts...> Func>
+  constexpr auto apply(Func func) const -> decltype(auto) {
```

Comment:
Feels like these should not be specific to just `meta::Set`. Maybe an algorithm library, or a parent class.

Reply:
Done.